connector/manifest.xml

```diff
@@ -10,24 +10,36 @@
     <vendor name="vendor"/>
     <driver name="driver"/>
     <customizations>
-      <customization name="CAP_SELECT_INTO" value="yes"/>
-      <customization name="CAP_SELECT_TOP_INTO" value="yes"/>
+      <!-- https://tableau.github.io/connector-plugin-sdk/docs/capabilities -->
+      <!-- Metadata -->
+      <customization name="CAP_QUERY_WHERE_FALSE_METADATA" value="yes"/>
+      <!-- Temporary Tables -->
       <customization name="CAP_CREATE_TEMP_TABLES" value="no"/>
+      <customization name="CAP_SELECT_INTO" value="no"/>
+      <customization name="CAP_SELECT_TOP_INTO" value="no"/>
+      <!-- String Splits -->
+      <customization name="CAP_SUPPORTS_SPLIT_FROM_LEFT" value="no"/>
+      <customization name="CAP_SUPPORTS_SPLIT_FROM_RIGHT" value="no"/>
+      <!-- Initial SQL-->
+      <customization name="CAP_SUPPORTS_INITIAL_SQL" value="yes"/>
+      <!-- Query -->
+      <customization name="CAP_QUERY_BOOL_IDENTIFIER_TO_LOGICAL" value="yes"/>
       <customization name="CAP_QUERY_BOOLEXPR_TO_INTEXPR" value="no"/>
+      <customization name="CAP_QUERY_CASE_OUT_NO_BOOL_OPS" value="no"/>
+      <customization name="CAP_QUERY_GROUP_BY_ALIAS" value="no"/>
       <customization name="CAP_QUERY_GROUP_BY_BOOL" value="yes"/>
-      <customization name="CAP_QUERY_GROUP_BY_DEGREE" value="yes"/>
+      <customization name="CAP_QUERY_HAVING_UNSUPPORTED" value="no"/>
+      <customization name="CAP_QUERY_INCLUDE_GROUP_BY_COLUMNS_IN_SELECT" value="no"/>
+      <customization name="CAP_QUERY_JOIN_PUSH_DOWN_CONDITION_EXPRESSIONS" value="no"/>
```

Comment:
I was basing this off the `schemas` term which I am disabling in the connectionMetadata [here](https://github.com/mongodb/mongo-tableau-connector/blob/a33b5cb2170b42b2fcaa1adaa71a4e8048d09d2e/connector/connectionMetadata.xml#L7).  I haven't found a scenario that tests this as the Tableau connection is based on one database.  
I can set this to `yes` as MongoSQL supports cross-database-lookup.

Reply:
Hmmm, maybe we don't need to set it to `yes` if we have `schemas` disabled 🤔  I'm not really sure.

Are you able to try to test a custom query like `SELECT * FROM db1.foo JOIN db2.bar`? or does that not work for various other reasons? We could always leave this omitted for now and investigate it in the future if we run into cross-db lookup problems!

Reply:
We've already filed a few follow-up tickets from the PR; this could certainly be another follow-up investigation!

Reply:
Sounds good, I've filed [SQL-754](https://jira.mongodb.org/browse/SQL-754)

The custom query with the different databases works through the Custom SQL option.  That is similar to running it through the ADL shell.  As for what Tableau changes (in terms of the query they run or options enabled/disabled) with that capability I'm not sure.  This can be part of the investigation ticket.

Reply:
> The custom query with the different databases works through the Custom SQL option.

Perfect. That's all I was really concerned about anyway!